src/saga_exec.rs

```diff
@@ -1677,89 +1659,42 @@ impl<UserType: SagaType> ActionContext<UserType> {
     pub fn lookup<T: ActionData + 'static>(
         &self,
         name: &str,
+        instance_id: u16,
```

Comment:
I still find the `instance_id` thing confusing...I'll poke at this a bit more to either come up with a concrete suggestion or some questions.

Reply:
🎉 `instance_id` is now a thing of the past thankfully. Very happy to get rid of it! Thanks for the changes @davepacheco.